asv_bench/benchmarks/hdfstore_bench.py

```diff
@@ -90,6 +90,15 @@ def time_query_store_table(self):
         stop = self.df2.index[15000]
         self.store.select('table', where="index > start and index < stop")
 
+    def time_store_repr(self):
+        repr(self.store)
```

Comment:
this may not show much as there is only a couple of nodes 

create a new store that has an example like your issue (but use only like 50 nodes)